bin/middlewares/unsupported_browser.js

```diff
@@ -1,10 +1,24 @@
+function isOldIE(userAgent) {
+  return userAgent.match('Trident'); // Test for IE <= 11
+}
+
+function isOldiOS(userAgent) {
+  if (/iP(hone|od|ad)/.test(userAgent)) {
+    const v = userAgent.match(/OS (\d+)_(\d+)_?(\d+)?/);
+    if (v && v[1]) {
+      return parseInt(v[1], 10) < 10; // test if major iOS version is < 10
+    }
+  }
+}
```

Comment:
Let's add a `return false;` here so it's clearer and doesn't just rely on the function returning `undefined`.